src/build.ts

```diff
@@ -92,6 +92,7 @@ export async function build(
 
   // For cache-breaking we rename most assets to include content hashes.
   const aliases = new Map<string, string>();
+  const plainaliases = new Map<string, string>();
```

Comment:
Another style question: I don't really like this `plainaliases` Map, but I didn't like either the idea of mixing the keys for aliases (with a ?hash) and plain aliases (without the ?hash) in the same `aliases` Map.